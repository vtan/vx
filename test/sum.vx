(
  (let i 100)
  (let sum 0)
  (while i (
    (set sum (+ sum i))
    (set i (- i 1))))

  (
    (let write 1)
    (let stdout 1)
    (let _ (syscall write stdout (addr-of sum) (* 2 8)))
  )
)
