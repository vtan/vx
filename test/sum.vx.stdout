00000000: ba13 0000 0000 0000 0000 0000 0000 0000  ................
